bokeh/models/tests/test_layouts.py

```diff
@@ -76,3 +76,11 @@ def check_widget_box_children_prop(layout_callable):
 def test_WidgetBox():
     check_props(WidgetBox())
     check_widget_box_children_prop(WidgetBox)
+
+def test_LayoutDOM_css_classes():
+    m = LayoutDOM()
+    assert m.css_classes == []
+    m.css_classes = ['foo']
+    assert m.css_classes == ['foo']
+    m.css_classes = ('bar', )
```

Comment:
I don't see a point in supporting tuples here or in general for `Seq`s.

Reply:
I can easily imagine defining a sequence of classes in tuples rather than lists, being able to assign from that directly is a nicety. But in this case also avoids changing the previous semantics.

Reply:
> But in this case also avoids changing the previous semantics.

That's definitively convenient. Let's leave it as-is and later change the semantics altogether, especially if we plan to maintain a decent level of compatibility between bokehjs typescript APIs and Python.

Reply:
What has to remain compatible and consistent are the property types and their default values. Adding language-appropriate convenience on top of that, e.g. accepting tuples, which are common in Python, for direct assignment, is completely orthogonal to Python/BokehJS compatibility.

Reply:
@mattpap are you requesting a change or are you ok with merging?

Reply:
No, OK with this.